astroquery/eso/core.py

```diff
@@ -640,43 +642,78 @@ def _print_help(self, url, instrument, cache=True):
         resp = self._request("GET", url, cache=cache)
         doc = BeautifulSoup(resp.content, 'html5lib')
         form = doc.select("html body form pre")[0]
-        for section in form.select("table"):
-            section_title = "".join(section.stripped_strings)
-            section_title = "\n".join(["", section_title,
-                                       "-" * len(section_title)])
-            result_string.append(section_title)
-            checkbox_name = ""
-            checkbox_value = ""
-            for tag in section.next_siblings:
-                if tag.name == u"table":
-                    break
-                elif tag.name == u"input":
-                    if tag.get(u'type') == u"checkbox":
-                        checkbox_name = tag['name']
-                        checkbox_value = u"[x]" if ('checked' in tag.attrs) else u"[ ]"
+
+        if instrument in ('feros', 'harps'):
+            for section in form.select("fieldset"):
+                section_title = "".join(section.stripped_strings.next())
```

Comment:
This part of the code doesn't have test coverage that's been checked on travis, but this line should fail with python3.